datasync/chaindatafetcher/metrics.go

```diff
@@ -0,0 +1,18 @@
+package chaindatafetcher
+
+import "github.com/rcrowley/go-metrics"
+
+var (
+	checkpointGauge = metrics.NewRegisteredGauge("chaindatafetcher/checkpoint/gauge", nil)
+
+	txInsertionTimeGauge            = metrics.NewRegisteredGauge("chaindatafetcher/insertion/tx/gauge", nil)
+	tokenTransferInsertionTimeGauge = metrics.NewRegisteredGauge("chaindatafetcher/insertion/tokentransfer/gauge", nil)
+	contractsInsertionTimeGauge     = metrics.NewRegisteredGauge("chaindatafetcher/insertion/contracts/gauge", nil)
+	tracesInsertionTimeGauge        = metrics.NewRegisteredGauge("chaindatafetcher/insertion/traces/gauge", nil)
```

Comment:
Why some are singular and others are plural?

Reply:
Applied. fa42787378437f56b89c9f5203bbf060bee5f285